opengrok-indexer/src/main/java/org/opengrok/indexer/history/RCSRepository.java

```diff
@@ -47,6 +44,16 @@
 
     private static final long serialVersionUID = 1L;
 
+    /**
+     * the property name used to obtain the command to get annotation for this repository
```

Comment:
Javadoc should have sentences - first sentence will be brief info. So I would start with upper case letter and end with `.`. CheckStyle has some checks for this. We can enable them later if the need arises.